messages/sfdxurl.store.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "authorize an org using an SFDX auth URL\nAuthorize a Salesforce org using an SFDX auth URL stored within a file. The file must have the format \"%s\" or \"%s\".\nThe file must contain only the URL or be a JSON file that has a top-level property named sfdxAuthUrl.\nUse this command to get the SFDX auth URL for a Dev Hub org you have already authorized:\n\n    $ sfdx force:org:display -u <DevHub> --verbose",
+  "description": "authorize an org using an SFDX auth URL\nAuthorize a Salesforce org using an SFDX auth URL stored within a file. The file must have the format \"%s\" or \"%s\".\nThe file must contain only the URL or be a JSON file that has a top-level property named sfdxAuthUrl or be a JSON file with the same format as the `sfdx force:org:display --verbose --json` command.\nUse this command to get the SFDX auth URL for a Dev Hub org you have already authorized:\n\n    $ sfdx force:org:display -u <DevHub> --verbose",
```

Comment:
Two edits:

1) Change the word "file" in the sentence that starts "The file must have the format..." to "URL".  (right?)

2) Replace the sentence that starts "The file must contain..." with this (i tried to add the right formatting, but you'll probably have to tweak it a bit):

"You have three options when creating the auth file. The easiest option is to redirect the output of the `sfdx force:org:display --verbose --json' command into a file. The resulting JSON file contains the sfdxAuthUrl property inside of a results object. For example:\n\n

sfdx force:org:display --verbose --json > authFile.json\n
sfdx auth:sfdxurl:store -f authFile.json\n\n

You can also create a JSON file that has a top-level property named sfdxAuthUrl whose value is the auth URL. Finally, you can create a non-JSON file that includes just the URL and nothing else. "

Reply:
Sounds good! How does this look? @amphro thoughts?
![image](https://user-images.githubusercontent.com/1084688/111223894-1f42a180-85b4-11eb-86b3-30df341f6aeb.png)